tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenCreateIndexTest.cs

```diff
@@ -24,16 +24,19 @@ namespace MongoDB.Driver.Tests.JsonDrivenTests
     public sealed class JsonDrivenCreateIndexTest : JsonDrivenCollectionTest
     {
         // private fields
-        private CreateIndexModel<BsonDocument> _createIndexModel;
+        private IndexKeysDefinition<BsonDocument> _indexKeys;
+        private string _indexName;
         private IClientSessionHandle _session;
 
-        // public constructors
-        public JsonDrivenCreateIndexTest(IMongoCollection<BsonDocument> collection, Dictionary<string, object> objectMap)
-            : base(collection, objectMap)
+        // properties
+        private CreateIndexModel<BsonDocument> CreateIndexModel
         {
+            get { return new CreateIndexModel<BsonDocument>(_indexKeys, new CreateIndexOptions { Name = _indexName }); }
```

Comment:
Should this be a lazy value instead of creating an instance each time the property is called?